wpilibcExamples/src/main/cpp/examples/StateSpaceArm/cpp/Robot.cpp

```diff
@@ -110,7 +110,7 @@ class Robot : public frc::TimedRobot {
     // Sets the target position of our arm. This is similar to setting the
     // setpoint of a PID controller.
     frc::TrapezoidProfile<units::radians>::State goal;
-    if (m_joystick.GetRightBumper()) {
+    if (m_joystick.GetRightBumperButton()) {
```

Comment:
So we're making a breaking change on a few names?  Let's make sure we capture that in documentation.

Reply:
The old names are still around but are deprecated.